runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -53,27 +50,7 @@
 		builder.addEscape('"', "'");
 		SHELL_ESCAPE = builder.build();
 
-		final String OS = System
-			.getProperty("os.name", "generic")
-			.toLowerCase();
-
-		if ((OS.contains("mac")) || (OS.contains("darwin")))
-		{
-			DETECTED_OS = OSType.MacOS;
-		}
-		else if (OS.contains("win"))
-		{
-			DETECTED_OS = OSType.Windows;
-		}
-		else if (OS.contains("nux"))
-		{
-			DETECTED_OS = OSType.Linux;
-		}
-		else
-		{
-			DETECTED_OS = OSType.Other;
-		}
-
+		DETECTED_OS = OSType.getOSType();
```

Comment:
This variable can probably be removed and OsType can be used here directly instead.